content/platform/data-services-cloud-console-api/home.md

```diff
@@ -0,0 +1,133 @@
+
+![DSCC Automation](/img/dscc-api-value.png "API for Automation at Scale")
+
+## REST API
+
+The REST API is designed for customers, partners or any other external clients to consume these interface to achieve the objectives mentioned above. The Data Services Cloud console public API is specified based on the OpenAPI format version 3 ([OpenAPI 3.0 information](https://swagger.io/blog/news/whats-new-in-openapi-3-0/)).  The specification defines standard, language agnostic interface to RESTful API allowing the clients (both human and computer) to consume capabilities of DSCC services efficiently. The API definition is available for download in OpenAPI 3 yaml format at this link. 
+
+Some of the advantages to distribution in OpenAPI 3.0 format:
+
+1. Updates to the API can be generated in more agile manner where documentation are embedded as description to any endpoints, parameters, and many more such as contact information, license, terms of use.
+2. Consumer of this API can also consume this API in an agile manner using the converter from this openAPI format to any language that were used as part of the their automation or CI/CD workflow. (Please check <https://openapi.tools> for more information for the API tools to generate client code)
+
+## REST API Details
+
+Any customer can download this OpenAPI Specification (OAS) v3 definition of the DSCC from the following link:
+
+[Link to the API repository](https://docs.ccs.arubathena.com/)
+
+### Documentation and Tutorials
+
+The overview of the Data Services Cloud Console is available at [this link](https://www.hpe.com/us/en/storage/data-services-cloud-console.html#overview). There will be more blogs that will provide more information to help customer adopting this API with examples, code snippets and many other helpful information.
+
+### Documented Attributes
+
+* **API Name & Description** - Provide short description of the goal for this API with the supported HTTP request method (POST, GET, etc)
+* **API Path** - Provide detail URL path as the end-point to issue the API call
+* **API Parameter**s - 
+
+### Supported API Categories (Services)
+
+The API categories for DSCC grows in accordance to the expansion of the DSCC services that are going to released in future. Current services that are available as recorded today is:
+
+1. DataOps Manager
+2. Event Audit
+3. Task tracking
+4. Search
+5. Helps, Issues and announcement
+
+### Versioning
+
+The major version number will be provided in the resource path as v1 in this example:
```

Comment:
Just a note... when using "v" for versions, shouldn't it be capitalized in each instance?

Reply:
let me use the quotation marks to point to the literal v1 in the path of the URI